test/avocado/selenium-machines-basic.py

```diff
@@ -107,3 +107,71 @@ def testDelete(self):
+
+    # def testOverviewInfo(self):
+    #     self.create_vm()
+
+    #     mem_ui = self.wait_css('#vm-staticvm-memory', cond=visible).text
+    #     vcpu_ui = self.wait_css('#vm-staticvm-vcpus', cond=visible).text
+    #     cpu_type_ui = self.wait_css('#vm-staticvm-cputype', cond=visible).text
+    #     emulatedmachine_ui = self.wait_css('#vm-staticvm-emulatedmachine', cond=visible).text
+    #     boot_ui = self.wait_css('#vm-staticvm-bootorder', cond=visible).text
+    #     autostart_ui = self.wait_css('#vm-staticvm-autostart', cond=visible).text
+
+    #     xml_for_vm = ET.fromstring(self.machine.execute('virsh dumpxml staticvm'))
+    #     mem = str(int(xml_for_vm.find('.//memory').text) // 1024) + ' MiB'
+    #     vcpu_count = xml_for_vm.find('.//vcpu').text
+    #     cpu_type = xml_for_vm.find('.//cpu').attrib['mode'] + ' (' + xml_for_vm.find('.//cpu/model').text + ')'
+    #     emulatedmachine = xml_for_vm.find('.//os/type').attrib['machine']
+    #     boot = xml_for_vm.find('.//os/boot').attrib['dev']
+    #     autostart = self.machine.execute('virsh dominfo staticvm | grep -i autostart').split(" ")[-1].replace("\n","d")
+
+    #     self.assertEqual(mem, mem_ui)
+    #     self.assertEqual(vcpu_count, vcpu_ui)
+    #     self.assertEqual(cpu_type, cpu_type_ui)
+    #     self.assertEqual(emulatedmachine, emulatedmachine_ui)
+    #     self.assertTrue(('disk' if boot == 'hd' else 'unknown') in boot_ui)
+    #     self.assertEqual(autostart, autostart_ui)
+
+    def testCreate20Machines(self):
+        for i in range(0, 20):
+            self.create_vm_on_ui('testVM{}'.format(i))
+
+    def testDelVmWithStorage(self):
+        args = self.create_vm()
+
+        pool_path = args.get('poolPath',None)
+        if not pool_path:
+            self.log.error('no poolPath')
+        base_img = pool_path + '/cirros.qcow2'
+        back_img = pool_path + '/backup.qcow2'
+        attach_img = pool_path + '/attach.qcow2'
+        self.machine.execute('sudo cp {} {}'.format(base_img,back_img))
+
+        self.machine.execute(
+            'sudo qemu-img create -f raw {} 128M && sudo virsh pool-refresh {}'.format(attach_img, args.get('poolName')))
+        self.machine.execute("sudo virsh attach-disk {} {} vda".format('staticvm',attach_img))
+
+        self.click(self.wait_css('#vm-staticvm-disks', cond=clickable))
+        self.wait_css('#vm-staticvm-disks-vda-bus')
+
+        self.click(self.wait_css('#vm-staticvm-delete', cond=clickable))
+        self.click(self.wait_css("#vm-staticvm-delete-modal-dialog button.btn-danger", cond=clickable))
+
+        self.wait_dialog_disappear()
+        self.wait_css('#vm-staticvm-row', cond=invisible)
+        self.assertNotIn('staticvm', self.machine.execute('sudo virsh list --all'))
+        self.assertNotIn(args.get('image',None), self.machine.execute('sudo virsh vol-list default'))
+        self.assertNotIn(attach_img, self.machine.execute('sudo virsh vol-list default'))
```

Comment:
What about the back_img? Don't you want to check for it being deleted?
Also please don't rely on the checkboxes being set, you have to check the checkboxes status as well.